content/general/concepts/file-formats/file-formats.md

```diff
@@ -23,9 +23,40 @@ Files can also be converted to other formats, however, some features for data di
 
 File formats exist for different types of files. Some common file types and formats are listed below:
 
+### Programing File Formats
+
+|                                       File Format                                       | Extension | Description                                                                                                  |
+| :-------------------------------------------------------------------------------------: | :-------: | ------------------------------------------------------------------------------------------------------------ |
+| [C++](https://www.codecademy.com/resources/docs/cpp)                                    | **.cpp**  | Conatins a C++ program.                                                                                      |
```

Comment:
Instead of being in **bold**, these extensions should be in `back-ticks`.

~**.cpp**~ -> `.cpp`